cn_indian_payroll/cn_indian_payroll/overrides/declaration.py

```
@@ -495,27 +260,26 @@ def choose_regime(doc_id, employee,payroll_period,company,regime):
                 epf_amount_sum = min(sum(epf_amount), 150000) if epf_amount else 0
 
-                # frappe.msgprint(f"NPS Amount Sum: {nps_amount_sum}")
-                # frappe.msgprint(f"EPF Amount Sum: {epf_amount_sum}")
-
                 get_declaration = frappe.get_doc("Employee Tax Exemption Declaration", doc_id)
 
                 form_data = {
                     'nineNumber': round(nps_amount_sum),
                     'pfValue': round(epf_amount_sum),
-                    'nineteenNumber': 2400  # Fixed PT amount
+                    'nineteenNumber':round(pt_amount_sum)
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)